ci/edmtool.py

```diff
@@ -174,7 +177,7 @@ def install(runtime, toolkit, pillow, environment, source):
     # edm commands to setup the development environment
     commands = [
         "edm environments create {environment} --force --version={runtime}",
-        "edm install -y -e {environment} {packages}",
+        "edm --config {edm_config} install -y -e {environment} {packages}",
```

Comment:
Looks like the `edm env create ...` command also needs the config

Reply:
Well spotted. Thank you.